content/en/account_management/plan_and_usage/cost_details.md

```diff
@@ -15,31 +15,34 @@ further_reading:
 
 ## Overview
 
-Estimated Cost Summary and Cost Chargebacks help you understand your estimated month-to-date and historical Datadog costs.
+Cost Summary and Cost Chargebacks help you understand your estimated month-to-date, projected end-of-month and historical Datadog costs.
 
 You can break down your costs by sub-organization and by product to:
 - Allocate costs according to their source
 - Gain insight into how costs are tracking
 
 ### Permissions
 
-To view the Estimated Cost Summary and Cost Chargebacks data, you must be a Datadog Admin user.
+To view the Cost Summary and Cost Chargebacks data, you must be a Datadog Admin user.
 
-Alternately, roles with Billing Read (`billing_read`) and Usage Read (`usage_read`) [permissions][1] can view the Estimated Cost Summary and Cost Chargebacks data.
+Alternately, roles with Billing Read (`billing_read`) and Usage Read (`usage_read`) [permissions][1] can view the Cost Summary and Cost Chargebacks data.
 
 ## Cost summary
 
 Use the cost summary to:
 - View estimated month-to-date costs
+- View projected end-of-month costs
 - View cost trends within the month
 - Filter and group costs by product or sub-organization
 - View cumulative day-over-day costs
 
-### Estimated Cost Summary (parent organization)
+Projected end-of-month costs are calculated by applying current month projected usage data against your contracted rates. Projections are available around the 12th of the month and are updated daily. Because the costs are a prediction, the amount may differ from your finalized monthly cost.
```

Comment:
```suggestion
Projected end-of-month costs are calculated by applying the current month's projected usage data against your contracted rates. Projections are available around the 12th of the month and are updated daily. Because the costs are a prediction, the amount may differ from your finalized monthly cost.
```